personal/cezarypukownik/reports/Cwiczenia3.Rmd

```diff
@@ -0,0 +1,184 @@
+---
+title: "Cwiczenia 3"
+output: html_document
+author: 'Cezary Pukownik'
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(sf)
+library(spData)
+library(knitr)
+
+```
+
+
+```{r, include=FALSE}
+#53°01′20″N 18°36′40″E
+
+lnd_point = st_point(c(18.36, 53.01))                 # sfg object
```

Comment:
na przyszłość jest to dobra praktyka, aby używać "<-" zamiast "=" do przypisywania zmiennych. Sam nie raz robię ten błąd i pojawił się często w prezentacji, więc muszę ją poprawić.